fastlane/metadata/android/en-US/title.txt

```diff
@@ -1 +1 @@
-Catima - Loyalty Card & Ticket Manager
+Catima —Card Wallet for Loyalty Cards, Tickets, and Cupons 🎴
```

Comment:
I think this needs a space. Also Coupons instead of Cupons.

Always what's that last icon? I can't recognise it as anything meaningful.

Reply:
https://www.fileformat.info/info/unicode/char/1f3b4/index.htm
It has the quality of being red when rendered in that "rich-UTF-8" style, and it is a card.